ftva_lab_data/templates/view_items.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>Items</h1>
+
+<!-- HTMX will listen for triggers from enclosed controls,
+then issue GET requests to 'render_table' URL
+and replace content of #table-container with result -->
+<form class="d-flex justify-content-end" hx-get="{% url 'render_table' %}" hx-target="#table-container"
+    hx-trigger="change from:find select, keyup changed delay:300ms from:find input">
+    <div class="d-flex w-50 gap-2">
+        <select name="column" class="form-select">
+            <option value="">All columns</option>
+            {% for field, label in columns %}
+            <option value="{{ field }}">{{ label }}</option>
+            {% endfor %}
+        </select>
+
+        <input id="search-input" class="form-control" type="text" name="search" placeholder="Enter search term...">
+    </div>
```

Comment:
Can you add a "reset" button, or an `X` control of some sort?  Not critical and it may not fit with the HTML aesthetic, but search interfaces generally have a UI element to at least clear the search box.